download/get.py

```diff
@@ -181,9 +180,10 @@ def name_flow(self, uuid):
         rows = df.loc[df['uuid']==uuid, 'name']
 
         # Policy: if there are many rows, return first value
-        return rows.iloc[0]
-
-
+        try:
+            return rows.iloc[0]
+        except IndexError:
+            return('')
```

Comment:
Buena idea la de cachar este error. Si ocurre es porque
1. Se han creado nuevos flows que ya han sido sido corridos (aparecen en runs.csv), y 
2. No se ha actualizado la base de flows.csv

No creo que se deba ignorar el error sino acompañar del string
`"Flow name not found. Call GetFlows.export_flows() to refresh flows.csv"`
¿Cómo lo ves?